remoteappmanager/file_config.py

```diff
@@ -77,18 +70,18 @@ def __init__(self, *args, **kwargs):
             self.docker_host = "unix://var/run/docker.sock"
 
         self.tls_verify = (env.get("DOCKER_TLS_VERIFY", "") != "")
```

Comment:
It looks to me something that future developers might unknowingly change it.
Maybe adding docker/docker#22411 to the code as comment.  We don't need to do it here, we can log this as one of the auditing issues.